### PR TITLE
Add one time scans to the frontend

### PR DESCRIPTION
**Pull request by FestiveKyle**
Adds individual scan cards for one time scans. These components should be refactored for abstract use with ScanCard in the future.

image

image

image

### CODE REVIEW

File: frontend/src/client.js [hunk 1]
```diff
@@ -65,7 +100,10 @@ export const currentUserVar = makeVar({
 })
 
 const httpLink = createHttpLink({
-  uri: '/graphql',
+  uri:
+    process.env.NODE_ENV === 'production'
+      ? 'https://tracker.alpha.canada.ca/graphql'
+      : '/graphql',
```
Comment (sleepycat) [inline frontend/src/client.js:106]: Tracker is going to be deployed at https://tracker.canada.ca and is currently running at https://staging.tracker.canada.ca and https://tracker.ops.cyber.gc.ca as well as our GCP environment at https://tracker.alpha.canada.ca.
I think leaving this at `/graphql` is probably a good idea.

File: frontend/src/client.js [hunk 2]
```diff
@@ -80,7 +118,39 @@ const headersLink = setContext((_, { headers }) => {
   }
 })
 
+const httpLinkWithHeaders = headersLink.concat(httpLink)
+
+const wsLink = new WebSocketLink({
+  uri:
+    process.env.NODE_ENV === 'production'
+      ? 'wss://tracker.alpha.canada.ca/graphql'
+      : 'ws:localhost:3000/graphql',
```
Comment (sleepycat) [inline frontend/src/client.js:127]: Same as above.

**Review by sleepycat — CHANGES_REQUESTED** [review]
Looks good but we're baking some assumptions about urls in there! Update those and we're good!

**Review by sleepycat — APPROVED** [review]
Nice! :star_struck:

**Review by lcampbell2 — APPROVED** [review]

**Pull request merged**